modules/claims_api/spec/requests/v2/veterans/disability_compensation_request_spec.rb

```diff
@@ -4575,167 +4575,56 @@ def update_json_and_submit(updated_json_lambda)
         end
       end
     end
-  end
-
```

Comment:
@acovrig I am a little confused, the ticket details seems to imply these would mveo to a different file, but they are still in the same file.  Also, does removing this 'end' now nest this inside the `describe '#526'` block?

Reply:
Yes, the original plan was to move it as I thought they were PDF mapper specs, but they actually tests for the controller. I renamed the ticket to match. I did find a couple duplicate specs and removed them. I did intentionally move the PDF specs to be under 526 because they test a 526 controller method.

Reply:
@acovrig Oh I see this this now, ok ya I was confused with sync still being out but I suppose that is another Pr.  I see.